plugins/woocommerce/client/legacy/js/admin/woocommerce_admin.js

```diff
@@ -1,93 +1,134 @@
 /* global woocommerce_admin */
-( function( $, woocommerce_admin ) {
```

Comment:
Did you make any changes in this file? or is it only formatting?
If only formatting I think you are better off reverting this change.